package.json

```diff
@@ -1,9 +1,34 @@
 {
   "name": "wildlife-licencing-service",
   "type": "module",
+  "version": "13.5.0-alpha.0",
   "engines": {
     "node": ">=16.13.0"
   },
+  "workspaces": [
+    "./packages/*"
+  ],
+  "scripts": {
+    "test": "jest",
+    "test-clear": "jest --clearCache",
+    "test:only-changed": "jest --changedSince master",
+    "test:watch": "jest --watch",
+    "format": "standard",
+    "lint": "standard",
+    "lint:fix": "standard --fix",
+    "docker:build-base": "sh Docker-build-base.sh",
+    "docker:build-services": "docker-compose -f docker/services.build.yml build --no-cache",
+    "docker:start-services": "docker-compose -f docker/services.yml up -d",
+    "docker:stop-services": "docker-compose -f docker/services.yml down",
+    "docker:start-cloud": "docker-compose -f docker/cloud-services.yml up -d",
+    "docker:stop-cloud": "docker-compose -f docker/cloud-services.yml down",
+    "docker:start-proxy": "docker-compose -f docker/nginx/nginx-service.yml up -d ",
+    "docker:stop-proxy": "docker-compose -f docker/nginx/nginx-service.yml down",
+    "prerelease:patch": "npm version prepatch --preid=alpha",
+    "prerelease:minor": "npm version preminor --preid=alpha",
+    "prerelease:major": "npm version premajor --preid=alpha",
+    "postversion": "git push && git push --tags"
+  },
```

Comment:
Would really help to have readme updated to reflect these script changes and ideally write down the new build and deploy process

Reply:
Readme updated for the docker compose work. 

I can update the release reamdme here - https://github.com/DEFRA/wildlife-licencing-team

Once we are happy.